src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -26,6 +27,9 @@
 
     private static final int SLEEP_TIME = 2000;
     private static final String FXML = "MainWindow.fxml";
+    private static final int CLASSES_TAB = 0;
```

Comment:
Perhaps you can make use of your Enum here?
`private static final int CLASS_TAB = Tab.CLASSES.getValue();`